release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -202,6 +202,18 @@ For more information, see _Managing machines with the Cluster API_.
 [id="ocp-4-16-nodes_{context}"]
 === Nodes
 
+[id="ocp-4-16-must-gather-directory_{context}"]
+==== Information added to directory when using must-gather
```

Comment:
I think that Andrea's suggestion previously was closer to what would make this maximally clear. I'm not sure if the current phrase centers what is really happening here--I have to give it a think after I read it.